### DISCUSSION TITLE
Welcome to zxing-android-embedded Discussions!

### DISCUSSION THREAD

**Discussion by rkistner** [journeyapps/zxing-android-embedded#595] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. This is a place to:
  * Ask questions about usage of the library.
  * Share ideas for new features.

Any bugs in the library should still be posted in [Issues](https://github.com/journeyapps/zxing-android-embedded/issues).

**Comment by wiktor0159** [comment]
Hello, I'm using yours library like shown here: https://www.youtube.com/watch?v=u2pgSu9RhYo
Can you tell me how to change auto focus to static value? For example 15 cm from camera? Or change it to some other value but no auto-focus? Please, it's important for me.

Wiktor
